CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v1.4.0-M1 (Feb 2020)
```

Comment:
This one should be clearly marked as milestone (unstable/experimental/development/etc.) version.